src/main.c

```diff
@@ -0,0 +1,49 @@
+/******************************************************
+* Author : Amuleen Gulati 
+* Function : main
+* Application : This function takes sensor values as input and 
+* calls different functions to produce an output
+* Input : a .csv file containing temperature values of sensors
+* Output : a single temperature value
+*/
+
+#include<stdio.h>
+
+#include "..\include\sdm.h"
+#include "..\include\eigen.h"
+#include "..\include\principle_comp.h"
+#include "..\include\contribution.h"
+
+int main()
+{
```

Comment:
See convention for curly braces on CULearn.

Reply:
For all the functions, I have followed proper conventions. I just skipped this somehow, Fixed it.